Rakefile

```diff
@@ -3,7 +3,7 @@ require "rake/testtask"
 require_relative "test/rake_helper"
 
 task default: [:test]
-task test: "test:postgis"
+task test: "test:all"
```

Comment:
I'll look at it. I have good experience with this suite now that I maintain the CRDB adapter. It usually is flaky due to some tests modifying the db. Which they should not do if they are correctly changed for our use case. Do you have any further information?

Reply:
Just this : https://github.com/rgeo/activerecord-postgis-adapter/issues/378

Reply:
Thx, I plan on excluding using `minitest-exclude` and opening an issue for excluded tests

Reply:
@BuonOmo yeah I hadn't figured out how to replicate the exclusions like in the CRDB adapter, but we definitely need that here since some are failing (which is expected). But to @seuros 's point that's why I just left the postgis tests for now.

Reply:
Now we only have Postgis basic tests flaky. If one of you has time to look into it it would be much appreciated. Otherwise maybe we can skip them for now and make this or top priority issue, just so we can release this ?

Reply:
@BuonOmo so just to make sure I'm clear, there's a few `postgis` tests that were working and are now flakey?  I can try to work through some of them. Is it only in the CI?

Reply:
I noticed some of the tests failures are order related.
notes here: https://github.com/rgeo/activerecord-postgis-adapter/pull/405#issuecomment-2409240511